packages/pwa-buildpack/lib/Utilities/runEnvValidators.js

```diff
@@ -32,9 +32,21 @@ async function validateEnv(context, env) {
 
     const validationContext = { env, onFail, debug };
 
-    await bus
-        .getTargetsOf('@magento/pwa-buildpack')
-        .validateEnv.promise(validationContext);
+    try {
```

Comment:
@zetlen I am using a try-catch to avoid `Buildpack targets not available error` while using the CLI tools. I am assuming the targets are not available because we are not going through buildpack/webpack while using the CLI tools. To avoid failures, I am wrapping this in the try-catch. Is that fine/expected?